README.md

```diff
@@ -5,15 +5,15 @@ Regular status-go image needs to be wrapped with a layer that has comcast (with
 This is what Dockerfile for in this repository.
 
 ```bash
-$ docker build -f Dockerfile . -t statusteam/statusd-debug:latest
-$ docker build -f Dockerfile-boot . -t statusteam/bootnode-debug:latest
+$ docker build -f Dockerfile -t statusteam/statusd-debug:latest .
+$ docker build -f Dockerfile-boot -t statusteam/bootnode-debug:latest .
 ```
 
 Also we are using bootnode image in tests, but it is not wrapped at this point.
 
 Install go dependencies:
 ```bash
-$ go get -u github.com/golang/dep/cmd/dep
+$ curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
```

Comment:
why not go get ?

Reply:
Well, this is the [recommended](https://github.com/golang/dep#installation) installation by dep guys, but we can keep go get